plugins/filter_kubernetes/kubernetes.c

```diff
@@ -229,6 +229,10 @@ static int pack_map_content(msgpack_packer *pck, msgpack_sbuffer *sbuf,
     msgpack_object v;
     msgpack_object root;
     struct flb_time log_time;
+    char *ptr_key = NULL;
+    char buf_key[256];
+    msgpack_object *key;
```

Comment:
I don't think it is a good practice, because `msgpack_object v;` already exists. Maybe I can turn variables `k` and `v` to pointers and change all calls that currently consider it as a `msgpack_object` ?

Reply:
I tried to improve this in a second commit.